client/src/api/index.js

```diff
@@ -34,16 +34,9 @@ export const addItemToInventory = async (inventory, item) => {
 
 export const deductCoin = async (userCoin, amount) => {
     const updatedCoin = userCoin - amount
-
-    if (updatedCoin === 0) {
-        await axios.put('/api/profile/info', {
-            coin: Math.floor(0.0001), // Set coin to null to remove the value
```

Comment:
Also, is this needed?

Reply:
It is not!

 I noted this bug at one point and did a quick adjustment to resolve miniscule floating point numbers in case that was that cause. Left this straggler behind, good catch!